feat(blog): blogpost about gov redeployment (#12256)
unlock-protocol/unlock: unlock-protocol-com/blog/governor-migration.md

---
title: Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version
publishDate: June 28, 2023
authorName: Clément Renaud

> image: /images/blog/ballot-box-image.png

Reviewer: ```suggestion
subTitle: The Unlock DAO deployed its governance contracts in 2021 using the then-current version of the Open Zeppelin Governor contracts. To take advantage of recent evolutions in the state-of-the-art, we believe now is the time for the Unlock DAO to redeploy a new version of the Open Zeppelin governor contracts and benefit from recent improvements.
description: The Unlock DAO deployed its governance contracts in 2021 using the then-current version of the Open Zeppelin Governor contracts. To take advantage of recent evolutions in the state-of-the-art, we believe now is the time for the Unlock DAO to redeploy a new version of the Open Zeppelin governor contracts and benefit from recent improvements.
image: /images/blog/ballot-box-image.png
```

---
title: Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version
publishDate: June 28, 2023
authorName: Clément Renaud
image: /images/blog/ballot-box-image.png
---

> 

Reviewer: ```suggestion
```

---

> title: Proposed Upgrade of the Unlock DAO Governor Contract 

Reviewer: ```suggestion
title: Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version
```

> title: Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version

Reviewer 2: ```suggestion
title: Proposed Upgrade of the Unlock DAO Governor Contract 
```

publishDate: June 28, 2023
authorName: Clément Renaud
image: /images/blog/ballot-box-image.png
---

> # **Proposed Upgrade of the Unlock DAO Governor Contract to the Most Recent Version**

Reviewer: ```suggestion
```

We have redeployed [a new version](https://etherscan.io/address/0x440d9D4E66d39bb28FB58729Cb4D3ead2A595591#code) of the governor contract and bumped from Open Zeppelin 4.4.2 to 4.9.0. This new contract will now be upgradeable so we can follow future releases without having to change its address.

## **The king is dead, long live the king!**

For the DAO, a change in governor requires two actions: canonize the new one and retire the old one. The vehicle for that process will be [a proposal](https://www.tally.xyz/gov/unlock/proposal/36208249270120864100503453462134662510103434369621143761091232235939585571890) containing the instructions to make the change. Once executed, this DAO popemobile will transport the voting system to its new location.

As a user/voter, there won’t be any change in delegations or voting process. The only apparent change is that the previous proposals will no longer appear in the Tally UI or be visible from the new governor contract.

## **What to do now**

> - [Delegate your votes](https://unlock-protocol.com/guides/delegation/) (you **must** delegate in order to participate in onchain votes, including self-delegation)

Reviewer 2: ```suggestion
-  If you own any amount of UDT, the Unlock protocol's governance tokens, [delegate your votes](https://unlock-protocol.com/guides/delegation/) (you **must** delegate to participate in onchain votes, including self-delegation)
```

- Head to [the proposal](https://www.tally.xyz/gov/unlock/proposal/36208249270120864100503453462134662510103434369621143761091232235939585571890)
- Cast your vote!

> The voting period ends on **xxx July 2023.** At the end of the voting period, and if the proposal to upgrade passes, the proposal will be queued and executed and the change will become effective.

Reviewer 2: ```suggestion
The voting period ends on **July 11th 2023**. At the end of the voting period, and if the proposal to upgrade passes, the proposal will be queued and executed and the change will become effective.
```
